scss/_patterns_grid.scss

```diff
@@ -190,40 +191,51 @@
   }
 
   // Common grid patterns
+  $col-50-medium: calc($grid-columns-medium / 2);
+
+  $col-25: calc($grid-columns / 4);
+  $col-50: calc($grid-columns / 2);
+  $col-75: calc(($grid-columns / 4) * 3);
 
   .row--50-50 {
     @extend %vf-row;
 
-    .#{$grid-column-prefix} {
+    > .col {
       @include vf-grid-column($grid-columns-small);
 
       @media (min-width: $threshold-4-6-col) {
-        @include vf-grid-column(calc($grid-columns-medium / 2));
+        @include vf-grid-column($col-50-medium);
       }
 
       @media (min-width: $threshold-6-12-col) {
-        @include vf-grid-column(calc($grid-columns / 2));
+        @include vf-grid-column($col-50);
       }
     }
   }
 
   .row--25-75 {
     @extend %vf-row;
 
-    .#{$grid-column-prefix} {
+    > .col {
       @include vf-grid-column($grid-columns-small);
 
       @media (min-width: $threshold-4-6-col) {
         @include vf-grid-column($grid-columns-medium);
       }
 
       @media (min-width: $threshold-6-12-col) {
-        &:nth-child(1) {
-          @include vf-grid-column(calc($grid-columns / 4));
+        &:nth-of-type(1) {
+          @include vf-grid-column($col-25);
+        }
+
+        &:nth-of-type(2) {
+          @include vf-grid-column($col-75);
         }
 
-        &:nth-child(2) {
-          @include vf-grid-column(calc($grid-columns / 4 * 3));
+        // if there is only one column we offset it to the right
+        &:only-of-type {
```

Comment:
Just noticed that, this is pretty cool!

I'm not super sure it's clear that it does that, but before we have any better implementation of the offset we can keep that. I like the simplicity of it, even if not super explicitly clear that this is gonna work this way. Especially that no other do this. But maybe they can?